docs/workflow/trimming/feature-switches.md

```diff
@@ -13,6 +13,7 @@ configurations but their defaults might vary as any SDK can set the defaults dif
 | EnableUnsafeBinaryFormatterSerialization | System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization | BinaryFormatter serialization support is trimmed when set to false |
 | EventSourceSupport | System.Diagnostics.Tracing.EventSource.IsSupported | Any EventSource related code or logic is trimmed when set to false |
 | InvariantGlobalization | System.Globalization.Invariant | All globalization specific code and data is trimmed when set to true |
+| PredefinedCulturesOnly | System.Globalization.PredefinedCulturesOnly |  Don't allow creating a made-up cultures when the platform does not carry data for it |
```

Comment:
`a made-up cultures` isn't grammatically correct.

How about 

```suggestion
| PredefinedCulturesOnly | System.Globalization.PredefinedCulturesOnly |  Don't allow creating a culture for which the platform does not have data |
```

?

Reply:
> Don't allow creating a culture for which the platform does not have data

feeling something missing. may be adding word `for` at the end make it more understandable.